### PR TITLE
converting sign users out guide to single page

### PR DESCRIPTION
**Pull request by chrisdmills-okta**
## Description:
- **What's changed?** https://developer.okta.com/docs/guides/sign-users-out/aspnet/before-you-begin/ coverted to a single page guide
- **Is this PR related to a Monolith release?** No
- Redirects tested on staging, 11/19/21. Redirects work fine, as expected.

### Resolves:

* [OKTA-438732](https://oktainc.atlassian.net/browse/OKTA-438732)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/guides/sign-users-out/main/index.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+## Sign users out of Okta
+
+Sign users out of Okta by ending their session on the Okta Authorization Server.
+
+<StackSelector snippet="remotesignout" noSelector/>
+
+## Sign users out of your app
+
+Sign users out your application by ending their local session. This signs the user out of your app, but doesn't [sign the user out of Okta](#sign-users-out-of-okta).
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/sign-users-out/main/index.md:72]:
```suggestion
Sign users out of your application by ending their local session. This signs the user out of your app, but doesn't [sign the user out of Okta](#sign-users-out-of-okta).
```

File: packages/@okta/vuepress-site/docs/guides/sign-users-out/main/index.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+
+## Next steps
+
+You should now understand how to sign users out of your app and out of Okta.
+
+Next, you might want to configure self-service registration for your org:
+
+* [Configure self-service registration](/docs/guides/set-up-self-service-registration/)
+
+Read more on customizing your org:
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/sign-users-out/main/index.md:88]: should this section be "See Also"?

**Review by susanharper-okta — COMMENTED** [review]
couple things/questions

File: packages/@okta/vuepress-site/docs/guides/sign-users-out/main/index.md [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+
+* An Okta Developer Edition organization. Don't have one? [Create one for free](https://developer.okta.com/signup).
+* An app that can sign in to Okta. To create your own, see the following guides:
+  * [Sign users in to your web app](/docs/guides/sign-into-web-app/)
+  * [Sign users in to your mobile app](/docs/guides/sign-into-mobile-app/)
+  * [Sign users in to your single-page app](/docs/guides/sign-into-spa/)
+
+**Sample code**
+
+* Android — see our [Android sample repo](https://github.com/okta/samples-android)
```
[the remark this answers is not in the document]
Comment (chrisdmills-okta) [reply packages/@okta/vuepress-site/docs/guides/sign-users-out/main/index.md:28]: I'm using liternal em-dash characters (created on mac keyboard using Shift + Alt + -), but I'm happy to use the HTML entity reference instead, as it is easier to spot. I'll change those in my next commit.

File: packages/@okta/vuepress-site/docs/guides/sign-users-out/main/index.md (hunk 2 above)
Comment (chrisdmills-okta) [reply packages/@okta/vuepress-site/docs/guides/sign-users-out/main/index.md:88]: I think so; I've updated the title and adjusted the text a bit to suit. Thanks!

**Comment by chrisdmills-okta** [comment]
Thanks for the review @susanharper-okta ! I've made all the fixes you suggested, and squashed the merge conflicts.

File: packages/@okta/vuepress-site/conductor.yml [hunk 4]
```diff
@@ -2765,6 +2765,26 @@ redirects:
     to: /docs/guides/unlock-mobile-app-with-biometrics/ios/main/#retrieve-refresh-token
   - from: /docs/guides/unlock-mobile-app-with-biometrics/ios/next-steps/index.html
     to: /docs/guides/unlock-mobile-app-with-biometrics/ios/main/
+  - from: /docs/guides/sign-users-out/-/before-you-begin
+    to: /docs/guides/sign-users-out/-/main/
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/conductor.yml:2769]:
```suggestion
    to: /docs/guides/sign-users-out/
```

File: packages/@okta/vuepress-site/conductor.yml [hunk 5]
```diff
@@ -2765,6 +2765,26 @@ redirects:
...
+  - from: /docs/guides/sign-users-out/-/define-signout-callback
+    to: /docs/guides/sign-users-out/-/main/#define-the-sign-out-callback
+  - from: /docs/guides/sign-users-out/-/sign-out-of-okta
+    to: /docs/guides/sign-users-out/-/main/#sign-users-out-of-okta
+  - from: /docs/guides/sign-users-out/-/sign-out-of-your-app
+    to: /docs/guides/sign-users-out/-/main/#sign-users-out-of-your-app
+  - from: /docs/guides/sign-users-out/-/next-steps
+    to: /docs/guides/sign-users-out/-/main/#see-also
+  - from: /docs/guides/sign-users-out/-/before-you-begin/index.html
+    to: /docs/guides/sign-users-out/-/main/
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/conductor.yml:2779]:
```suggestion
    to: /docs/guides/sign-users-out/
```

**Review by susanharper-okta — APPROVED** [review]
two final nits...I'll approve so that I don't hold you up.

**Pull request merged**